app/views/shared/tas/_juno.html.slim

```diff
@@ -0,0 +1 @@
+| Hi, I'm Juno. I'm a second year intended CS major (also thinking of cog sci). In my spare time I like to produce music, go hiking, play smash (melee), play guitar, watch reviews of cars I'll probably never be able to afford, and once in a while I paint. I think Rails is cool because it gives you all the tools you need to turn an idea into reality. Hope you enjoy the course!
```

Comment:
Extra space (just hit enter) after this line (that'll make that little red warning sign go away)!